include/avif/internal.h

```diff
@@ -373,6 +374,7 @@ typedef avifResult (*avifCodecEncodeImageFunc)(struct avifCodec * codec,
                                                int quantizer,
                                                avifEncoderChanges encoderChanges,
                                                avifAddImageFlags addImageFlags,
+                                               avifBool disableLaggedOutput,
```

Comment:
Nit/optional: let `addImageFlags` be the last input parameter (a common convention).

Reply:
Done.